cmd/create/idp/cmd.go

```diff
@@ -29,9 +29,14 @@ import (
 	"github.com/openshift/moactl/pkg/interactive"
 	"github.com/openshift/moactl/pkg/logging"
 	"github.com/openshift/moactl/pkg/ocm"
+	"github.com/openshift/moactl/pkg/reporter"
 	rprtr "github.com/openshift/moactl/pkg/reporter"
 )
 
+type IdentityProvider interface {
+	Name() string
+}
```

Comment:
Is this interface used for the tests/mocks? Otherwise it's probably unnecessary (that's just me nitpicking, definitely not a blocker)

Reply:
as I know @NautiluX, this is to make that string more meaningful.. the usage in the code does not require the interface

Reply:
to be able to mock the `IdentityProvider` from the OCM lib used, it's necessary to decouple it, yes. I don't like that we have to convert the array because you can't automatically use a `[]SomeIf` as `[]*SomeType` (SomeType implements SomeIf) unfortunately. The interface itself is not a bad thing, but I don't like the conversion logic. Happy to change it if somebody has a better idea :)

Reply:
Maybe create a type alias?
type IdentityProvider string

See example https://github.com/openshift/moactl/blob/45d6607f2ef01db3b1ce9ce6c0631832b644c1ca/OWNERS#L6

Reply:
No, this is fine. Let's leave it as is for now and get this bugfix merged.